plugin/postgres/plugin.go

```diff
@@ -154,8 +156,21 @@ func (p PostgresPlugin) Backup(endpoint ShieldEndpoint) error {
 	}
 
 	setupEnvironmentVariables(pg)
-	cmd := fmt.Sprintf("%s/pg_dumpall -c --no-password", pg.Bin)
-	DEBUG("Executing: `%s`", cmd)
+
+	// If user specified specific database to backup
+	db_name := pg.Database
+
+	cmd := ""
+	// Check if db_name is valid
+	if db_name != "" {
+		// Run dump all on the specified db
+		cmd = fmt.Sprintf("%s/pg_dump %s", pg.Bin, db_name)
```

Comment:
We're going to want to keep the `-c --no-password` flags for pg_dump here as well